.github/labeler.yml

```diff
@@ -0,0 +1,17 @@
+schedule:
+- any: ['src/components/SchedulePage/**/*', 'src/pages/schedule.js']
+
+homepage:
+- any: ['src/components/HomePage/**/*', 'src/pages/index.js']
+
+prizes:
+- any: ['src/components/PrizePage/**/*', 'src/pages/prizes.js']
+
+page:
+- any: ['src/components/MenuBar/**/*', 'src/components/Footer/**/*', 'src/components/layout.js']
+
+gallery:
+- any: ['src/components/GalleryPage/**/*', 'src/pages/gallery.js']
+
+infra:
+- any: ['.github/**/*', '!.github/FUNDING.yml']
```

Comment:
Let's also add these to `infra`
```
src/components/SvgImg.js
.editorconfig
.eslint*
.huskyrc.js
.nvmrc
.prettier*
.svgo*
gatsby-*.js
package.json
yarn.lock
```

And the following to `documentation`:
```
LICENSE*
*.md
.github/FUNDING.yml
```

Finally, make sure you add a new line to the end of files.